neutopia/src/chest.rs

```diff
@@ -21,6 +21,57 @@ impl Chest {
 
         Ok(())
     }
+
+    pub fn get_item_name(&self) -> String {
+        return match self.item_id {
+            0 => format!("{} bombs", self.arg),
+            1 => format!("Medicine"),
+            2 => format!("Firewand"),
+            3 => format!("Sky Bell"),
+            4 => format!("Wings"),
+            5 => format!("Moonbeam Moss level {}", self.arg),
+            6 => format!("Magic Ring"),
+            7 => format!("Placeholder"),
+            8 => match self.arg {
+                1 => format!("Starter Sword"),
+                2 => format!("Blue Sword"),
+                3 => format!("Purple Sword"),
+                4 => format!("Red Sword"),
+                _ => format!("Unknown Sword"),
+            },
+            9 => match self.arg {
+                1 => format!("Starter Armor"),
+                2 => format!("Blue Armor"),
+                3 => format!("Purple Armor"),
+                4 => format!("Red Armor"),
+                _ => format!("Unknown Armor"),
+            },
+            10 => match self.arg {
+                1 => format!("Wood Shield"),
+                2 => format!("Steel Shield"),
+                3 => format!("Fire Shield"),
+                4 => format!("Blue Shield"),
+                _ => format!("Unknown Shield"),
+            },
+            11 => format!("Falcon Shoes"),
+            12 => format!("Rainbow Drop"),
+            13 => format!("Book of Revival"),
+            14 => format!("Placeholder"),
+            15 => format!("Placeholder"),
+            16 => format!("Crystal Ball"),
+            17 => format!("Crypt Key"),
+            18 => format!("Medallion"),
```

Comment:
What do you think about naming these "Crypt 1 Medallion", "Crypt 2 Medallion", etc.?

Reply:
Yeah, that'd make it easier to read. Is it linearly correlated though? The positions of medallions in the shrine are all I can go on: https://github.com/konkers/neutopia-docs/pull/1/files